render/render.js

```diff
@@ -20,6 +20,7 @@ module.exports = function($window) {
 	}
 	function createNode(parent, vnode, hooks, ns, nextSibling) {
 		var tag = vnode.tag
+		vnode.reuse = true
```

Comment:
Mind dropping an underscore for this member and also creating an entry in the vnode constructor in `render/vnode.js`?